actions/readFile.go

```diff
@@ -0,0 +1,60 @@
+package actions
+
+import (
+	"geektrust/helpers"
+	"geektrust/models"
+	"strings"
+)
+
+//Init function starts the program
+func Init(path string) bool {
+	isFileExist := helpers.FileExists(path)
+	if isFileExist {
+		//This means file exists
+		allCommands := helpers.ReadFileFromPath(path)
+		if len(allCommands) > 0 {
+			//There are some commands
+			var validCommands [][]string
+			for _, commandLine := range allCommands {
```

Comment:
let's use shorter variable names for short-lived variables. 
https://github.com/golang/go/wiki/CodeReviewComments#variable-names

This can be implemented throughout the other files and functions in the code base too.

Reply:
Will this change hamper code readability by any chance?

Reply:
The convention in GO is that we use short name short lived variables like iteration indexes etc. 
From the Go's wiki:

"The basic rule: the further from its declaration that a name is used, the more descriptive the name must be. For a method receiver, one or two letters is sufficient. Common variables such as loop indices and readers can be a single letter (i, r). More unusual things and global variables need more descriptive names."

Here is a nice ppt that talks about naming conventions in go - https://talks.golang.org/2014